themes/twitter/twitter.scss

```diff
@@ -0,0 +1,177 @@
+// Twitter Theme for SweetAlert
+// By Tristan Edwards
+
+.sweet-overlay {  
+  background: rgba(41,47,51,0.9);
+}
+
+
+.sweet-alert {
+  $header-height: 40px;
+  $footer-height: 66px;
+  $text-color: #66757f;
+  $padding: 15px;
+
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  padding: $padding;
+  padding-top: $header-height + $padding;
+  text-align: right; // Align buttons
+  border-radius: 6px;
+  box-shadow: 0px 0px 0px 1px rgba(black, 0.11), 0px 6px 30px rgba(black, 0.14);
+
+  h2 {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: $header-height;
+    line-height: $header-height; 
+    font-size: 16px;
+    font-weight: 400;
+    color: #8899a6;
+    margin: 0;
+    color: $text-color;
+    border-bottom: 1px solid #e1e8ed;
+  }
+
+  p {
+    display: block;
+    text-align: center;
+    color: #66757f;
+    font-weight: 400;
+    font-size: 13px;
+    margin-top: 7px;
+  }
+
+  .sa-button-container {
+    background-color: #f5f8fa;
+    border-top: 1px solid #e1e8ed;
+    box-shadow: 0px -1px 0px white;
+    margin: -$padding;
+    margin-top: 0;
+  }
+  &[data-has-confirm-button=false][data-has-cancel-button=false] {
+    padding-bottom: 10px;
+    .sa-button-container {
+      display: none;
+    }
+  }
+
+  button {
+    border-radius: 2px;
+    box-shadow: none !important;
+    text-shadow: 0px -1px 0px rgba(black, 0.3);
+    margin: 17px 0px;
+    border-radius: 4px;
+    font-size: 14px;
+    font-weight: 600;
+    padding: 8px 16px;
+    position: relative;
+    &:focus, &.cancel:focus {
+      box-shadow: none !important;
+      &::before {
+        content: "";
+        position: absolute;
+        left: -5px;
+        top: -5px;
+        right: -5px;
+        bottom: -5px;
+        border: 2px solid #a5b0b4;
+        border-radius: 8px;
+      }
+    }
+
+    &.confirm {
+      background-color: #55acee !important;
+      background-image: linear-gradient(rgba(0,0,0,0),rgba(0,0,0,0.05));
+      -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#00000000, endColorstr=#0C000000)";
+      border: 1px solid #3b88c3;
+      box-shadow: inset 0 1px 0 rgba(255,255,255,0.15);
+      margin-right: $padding;
+      &:hover {
+        background-color: #55acee;
+        background-image: linear-gradient(rgba(0,0,0,0),rgba(0,0,0,0.15));
+        -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#00000000, endColorstr=#26000000)";
+        border-color: #3b88c3;
+      }
+    }
+    &.cancel {
+      color: #66757e;
+      background-color: #f5f8fa;
+      background-image: linear-gradient(#fff,#f5f8fa);
+      text-shadow: 0px -1px 0px white;
+      margin-right: 9px;
+      border: 1px solid #e1e8ed;
+      &:hover, &:focus:hover {
+        background-color: #e1e8ed;
+        background-image: linear-gradient(#fff,#e1e8ed);
+        -ms-filter: "progid:DXImageTransform.Microsoft.gradient(enabled=false)";
```

Comment:
Properties should be ordered background-color, background-image, border-color, -ms-filter<br>Avoid vendor prefixes.